src/build.ts

```diff
@@ -14,77 +12,82 @@ import { getDirname, getVersion } from "./module-specific.js";
 const version = getVersion();
 const _dirname = getDirname();
 
-const pipe = promisify(pipeline);
-
 export interface Platarch {
   name: string;
   filetype: string;
-};
+}
 
 // These must match the builds that go-tableland is automating
 const platarchs: Platarch[] = [
   {
     name: "darwin-amd64",
-    filetype: ".tar.gz"
+    filetype: ".tar.gz",
   },
   {
     name: "linux-amd64",
-    filetype: ".tar.gz"
+    filetype: ".tar.gz",
   },
   {
     name: "windows-amd64",
-    filetype: ".zip"
-  }
+    filetype: ".zip",
+  },
+  // TODO: add M1 build if/when that makes sense
 ];
 
 const binDirectory = join(_dirname, "..", "..", "bin");
 
-const go = async function () {
-  console.log(`installing validator binaries: ${platarchs.map(pa => `${pa.name}${pa.filetype}`).join(", ")} for version: ${version}`);
+const go = async function (): Promise<void> {
+  console.log(
+    `installing validator binaries: ${platarchs
+      .map((pa) => `${pa.name}${pa.filetype}`)
+      .join(", ")} for version: ${version as string}`
```

Comment:
Not sure exactly why, but the ts compiler is complaining unless I use `version as string` 😞

Reply:
This is because in new versions of ts, it doesn't like having anything other than a strict string in formatted strings. So you need to explicitly define it to be string.

Reply:
I tried defining the return type as a string, but I think that the cjs/ems module specific files aren't letting the ts compiler know.
ref: https://github.com/tablelandnetwork/js-validator/pull/6/files#diff-7203482ae83495c0be48d2b5a97a8e4294ed314141d5c047d9ade6347feb8f63